build-everything.proj

```diff
@@ -43,6 +43,10 @@
     <SetupProjects Include="setup/fsharp-setup-build.proj" />
   </ItemGroup>
 
+  <ItemGroup Condition="'$(BUILD_NUGET)'=='1'">
+    <NugetProjects Include="src/fsharp/fsharp.core.nuget/fsharp.core.nuget.proj" />
```

Comment:
Ugliest thing about dotnet programming is PascalCased assemblies and Properties, and camelCased fields.  Worst thing about unix is a case sensitive file System.